sigle/src/modules/editor/extensions/SlashCommand/CommandList.tsx

```diff
@@ -133,6 +157,26 @@ export const CommandList = forwardRef<CommandListRef, CommandListProps>(
             )}
           </CommandsListItem>
         ))}
+        <CommandsListSeparator />
+        <CommandsListLabel>Embeds</CommandsListLabel>
```

Comment:
If we were to merge that pr now, users would see the Twitter embed, I think this change should go in the twitter pr or under a flag :)

Reply:
ah yes sure I'll remove and add it to the twitter pr 👍